source/common/json/config_schemas.cc

```diff
@@ -72,6 +72,78 @@ const std::string Json::Schema::CLIENT_SSL_NETWORK_FILTER_SCHEMA(R"EOF(
   }
   )EOF");
 
+const std::string Json::Schema::HTTP_CONN_NETWORK_FILTER_SCHEMA(R"EOF(
+  {
+    "$schema": "http://json-schema.org/schema#",
+    "definitions" : {
+      "access_log": {
+        "type" : "object",
+        "properties" : {
+          "path" : {"type" : "string"},
+          "format" : {"type" : "string"},
+          "filter" : {"type" : "object"}
```

Comment:
I am going to add the access log schemas in another pr so I can update the example configs. I can extend this schema when I do that pr (after the http filters). Sounds good?